backend/app/views.py

```
@@ -1331,7 +1331,7 @@ def layerid_view(request, instance):
             return HttpResponse(headers={"HX-Trigger-After-Settle": json.dumps({"showDangerAlert": "Provide a layer URL."})})
 
         try:
-            layer_usage_result = utils.find_layer_usage(portal, layer_url_searched)
+            layer_usage_result = utils.service_layer_details(portal, layer_url_searched)
```

Ruff (0.14.10)

`utils` may be undefined, or defined from star imports

(F405)